content/2018-11-14-this-year-in-embedded-rust.md

```diff
@@ -0,0 +1,131 @@
++++
+title = "This year in embedded Rust"
+date = 2018-11-14
+draft = false
+in_search_index = true
+template = "page.html"
++++
+
+This year the [embedded WG] set out to build the solid foundation that the
+embedded Rust ecosystem requires to thrive. As we approach the date of the 2018
+edition release we reflect on our progress and share our achievements with you
+in this post.
+
+[embedded WG]: https://github.com/rust-embedded/wg
+
+## Embedded Rust on stable
+
+Stability -- AKA "my crate should not break when moving to a newer toolchain" --
+was the single most requested feature by the embedded community during the 2018
+roadmap planning phase.
+
+Embedded development has been tied to the nightly
+channel since its very beginning, and the people that have been doing embedded
+development for a while have endured many breaking changes. Thankfully, those
+days are over: Rust 1.30 marks the first stable release where you can build
+fully working embedded programs without relying on unstable features.
+
+Furthermore, we now have some [unit][] [tests] that check embedded code in the
```

Comment:
Extra `[]`?

Reply:
this is to avoid this being parsed as `[unit][tests]` which would render as a single word / broken link.